src/components/ebay-progress-bar/component.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
-    getValue(value, min) {
-        const parsedValue = parseInt(value, 10);
-        if (parsedValue <= min) {
-            return min + 1;
-        }
-        return parsedValue;
+  getValue(value, min) {
```

Comment:
Not sure why this file got changed from 4 spaces to 2...?

Reply:
@austingardner I think this is the only thing that needs addressing before merge

Reply:
Done. What happened was that prettier's default is 2 and prettier was run on this file. Now that we have it set to 4 I ran it again and it's back to where it was.